frontend/src/components/Post.js

```diff
@@ -50,7 +52,7 @@ class Post extends React.Component {
                     <div className="post">
                         <div className="image-wrapper">
                             <img src={`${global.config.BACKEND_URL}/picture/${this.props.id}.jpg`}
-                                 alt="insert post here"/>
+                                alt="insert post here" />
```

Comment:
remnant of old code, should probably look into removing the this

Reply:
The `this` is required in class component in order to access the variables.

Reply:
i was referring to the `alt="..."`